lib/game.rb

```diff
@@ -57,10 +57,60 @@ def valid_move(index)
     end
   end
 
+  def horizontal_win
```

Comment:
It's usually idiomatic to name methods like this (that just return a boolean) with a question mark, like this:

```ruby
def horizontal_win?
  # ...
```